quickevent/app/plugins/Event/src/eventplugin.cpp

```diff
@@ -937,34 +936,42 @@ bool EventPlugin::openEvent(const QString &_event_name)
 				}
 				else {
 					qfd::MessageBox::showError(fwk, tr("Open Database Error: %1").arg(conn.errorString()));
+					ok = false;
 				}
 			}
 			else {
 				qfd::MessageBox::showError(fwk, tr("Database file %1 doesn't exist.").arg(event_fn));
+				ok = false;
 			}
+
 		}
 	}
-	else {
-		qfError() << "Invalid connection type:" << static_cast<int>(connection_type);
-	}
 	if(ok) {
 		EventConfig *evc = eventConfig(true);
 		if(evc->dbVersion() < minDbVersion()) {
 			qfd::MessageBox::showError(fwk, tr("Event data version (%1) is too low, minimal version is (%2).\nUse: File --> Import --> Event (*.qbe) to convert event to current version.")
 									   .arg(qf::core::Utils::intToVersionString(evc->dbVersion()))
 									   .arg(qf::core::Utils::intToVersionString(minDbVersion())));
 			closeEvent();
-			return false;
+			ok = false;
 		}
-		if(evc->dbVersion() > minDbVersion()) {
+		else if(evc->dbVersion() > minDbVersion()) {
 			qfd::MessageBox::showError(fwk, tr("Event was created in more recent QuickEvent version (%1) and the application might not work as expected. Download latest QuickEvent is strongly recommended.")
 									   .arg(qf::core::Utils::intToVersionString(evc->dbVersion())));
 		}
+	}
+	if(ok) {
 		connection_settings.setEventName(event_name);
 		setEventName(event_name);
 		emit eventOpened(eventName());
 		//emit reloadDataRequest();
 	}
+	m_actEditStage->setEnabled(ok);
+	m_actOpenEvent->setEnabled(ok);
+	m_actEditEvent->setEnabled(ok);
```

Comment:
when I enter invalid event name and I cancel open dialog, then action `edit event` remains enabled

Reply:
if open dialog is canceled, `ok` is set to false

Reply:
my fault, I tried it in wrong guild directory